test/stablehlo/llama_model.py

```diff
@@ -259,11 +259,11 @@ def __init__(self, params: ModelArgs):
   def forward(self, tokens: torch.Tensor, start_pos: int):
     _bsz, seqlen = tokens.shape
     h = self.tok_embeddings(tokens)
-    # freqs_cis = self.freqs_cis[start_pos : start_pos + seqlen]
+    freqs_cis = self.freqs_cis[start_pos:start_pos + seqlen]
     # freqs_cis = self.freqs_cis[:seqlen] #(self.freqs_cis, 0, 0, seqlen)
-    freqs_cis = torch.index_select(
```

Comment:
nit: In the XLA execution workflow, we need to use `index_select` to avoid a cpu fallback. Otherwise the indexing operation will fallback to cpu. Call this out just in case we want the model code to be executed.

Reply:
yeah the llama_model2 version have the one using index_select